selenium/che-selenium-test/src/test/resources/suites/CheSuite.xml

```diff
@@ -886,18 +886,6 @@
                 </methods>
             </class>
             <class name="org.eclipse.che.selenium.refactor.types.TestFailTest"/>
-            <class name="org.eclipse.che.selenium.subversion.CommitTest"/>
```

Comment:
It have not been finally decided to remove support of SVN plugin yet https://github.com/eclipse/che/issues/5484#issuecomment-328075154

Reply:
@akorneta Pls, restore removed tests.